docs/docs/pyproject.md

```diff
@@ -173,10 +173,25 @@ If a VCS is being used for a package, the exclude field will be seeded with the
 include = ["CHANGELOG.md"]
 ```
 
+You can also use inline tables for `include`:
+
+```toml
+[tool.poetry]
+# ...
+include = [
+    { include = "tests", format = "sdist" },
+    { include = "for_wheel.txt", format = "wheel" }
+]
+```
+
 ```toml
 exclude = ["my_package/excluded.py"]
 ```
 
+!!!note
+
+    You can either use strings or inline tables for `include` but not both.
+
```

Comment:
This can be removed since the TOML 1.0.0rc1 specification now allows mixed-types arrays.